### PR TITLE
First pull request with changes to review

### PR DESCRIPTION
**Pull request by gfrancoarq**
Added all the "Hello Microverse" project requirements, including the html page, css, readme file, and the test.md ignored by git.

### CODE REVIEW

**Review by muneebulrehman — COMMENTED** [review]
Hi @gfrancoarq 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a PR that has no description.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by gfrancoarq** [comment]
"Thank you. I'll make that change."

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Lighthouse
+        run: npm install -g @lhci/cli@0.7.x
+      - name: Lighthouse Report
+        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
+  webhint:
```
Comment (1GORDON) [inline .github/workflows/linters.yml:21]: Great job @gfrancoarq 🤝

- [ ] Great job adding the `linters.yml` file. However, for this project, you need to add both the `.hintrc` file and the `.stylelintrc.json` file. Remember to add these two files to the root directory. Please find the correct [linter files in this link](https://github.com/microverseinc/linters-config/tree/master/html-css).

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @gfrancoarq ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflows/linters.yml (hunk 1 above)
Comment (gfrancoarq) [reply .github/workflows/linters.yml:21]: I added the files. I'll wait for your comments. Thank you!

File: README.md [hunk 2]
```diff
@@ -1,2 +1,58 @@
...
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```
Comment (BenMukebo) [inline README.md:36]: - [OPTIONAL] Make sure to remove all unnecessary titles since they are empty (Prerequisites, run tests Deployment ....)

**Review by BenMukebo — APPROVED** [review]
### Hi @gfrancoarq , 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**